test/private/test_nicolet_reading_onefile.m

```diff
@@ -0,0 +1,87 @@
+function test_nicolet_reading_onefile(path_to_load,nicoletFileName,asciiVerifyFileName,Fs,nChans,nSamples,nTrials,startDateTime)
```

Comment:
These functions (i.e. the ones in fieldtrip/test) should also be functional without input arguments, in order to serve as part of an automatically deployed batch of test functions, which should run on our local file system.
Would it be possible to initialize the input arguments (in case of nargin==0) to values that allow it to work with a prespecified file, and to share that particular file with us (or point to where it can be downloaded from)?

Reply:
I shared the data in a separate repo https://github.com/janbrogger/FieldTripNicoletTestData/ . The other testing function is callable without parameters: test_nicolet_reading . The function test_nicolet_reading_onefile is not meant to be called without parameters.